trappy/ftrace.py

```diff
@@ -88,6 +145,13 @@ def __init__(self, name="", normalize_time=True, scope="all",
         if normalize_time:
             self.normalize_time()
 
+        if self.__class__.disable_cache: return
+
+        self._create_trace_cache()
+        for trace_class in self.trace_classes:
+            csv_file = self._get_csv_path(trace_class)
+            trace_class.write_csv(csv_file)
+
```

Comment:
That actually got me thinking - would Trappy be a bad OS citizen by dropping these ".cache" directories all around people's filesystems? E.g. on Linux should we use `/var/cache`? That would introduce platform-specific stuff so maybe it's not worth the hassle.